ui/components/src/ui-components/propertygrid/dataproviders/filterers/CompositePropertyDataFilterer.ts

```diff
@@ -31,10 +32,28 @@ export class CompositePropertyDataFilterer extends PropertyDataFiltererBase {
 
   public get isActive() { return this._leftFilterer.isActive || this._rightFilterer.isActive; }
 
-  public async matchesFilter(node: PropertyRecord, parents: PropertyRecord[]): Promise<PropertyDataFilterResult> {
+  public async recordMatchesFilter(node: PropertyRecord, parents: PropertyRecord[]): Promise<PropertyDataFilterResult> {
```

Comment:
these two methods are nearly identical - need to avoid code duplication

Reply:
Extracted some code